src/pdfviewer/PDFDocument.h

```diff
@@ -675,8 +682,10 @@ private slots:
 
     QStatusBar *statusbar;
     QToolBar *toolBar;
+    QToolBar *tbPdfView;
```

Comment:
unused, please remove

Reply:
indeed, and change to definition of setupToolBar must also be removed. This is done.